<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelSynthese</class>
 <widget class="QWidget" name="PanelSynthese">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_main">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="m_pListYears">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableWidget" name="m_pTableInput">
       <column>
        <property name="text">
         <string>item</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>2</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="m_pTableOutput"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnExport">
            <property name="toolTip">
             <string>Save  synthesis into CSV file</string>
            </property>
            <property name="text">
             <string>EXPORT</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/ico/png/fileopen.png</normaloff>:/ico/png/fileopen.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpdateItem">
            <property name="toolTip">
             <string>Generate the report</string>
            </property>
            <property name="text">
             <string>UPDATE</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/ico/png/refresh-32.png</normaloff>:/ico/png/refresh-32.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
